### DISCUSSION TITLE
一些新设计

### DISCUSSION THREAD

**Discussion by zxbmmmmmmmmm** [HyPlayer/HyPlayer#313] [Ideas]
(希望)能够在3.0实现
## 主页
![1G)D 88@CV``9HVN 6Q 7IN](https://user-images.githubusercontent.com/96322503/205427853-c57b720e-6b02-477c-8427-de1989b8d64b.jpg)
##

**Comment by zxbmmmmmmmmm** [comment]
![image](https://user-images.githubusercontent.com/96322503/205442928-4a7b8b5d-98f5-4f9c-85e0-8f29ada927fe.png)
更改了SuggestBox的位置和一些微调

**Reply by kengwang** [reply]
Wow, beautiful!
